### PR TITLE
navbar staying on it's place after click on any link bug fixed

### LINKED ISSUE #4128: In mobile view After clicking on some links in the nav item the navbar stays on it's place.
### Description

In Mobile view whenever we clicked on any pages in the navbar the navbar stays on it's place .
### Expected Behavior

The expected behaviour should be the navbar got removed from the display because we click on the page to visit it but the navbar blocking our view
### Environment:
- Host OS: Android 
- Browser: Chrome 

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by shibu-g**
**Description**

This PR fixes #4128 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 7981e650c223878153cde0b475b30cfa40e11b28 at: https://6447bb85d87cb35915989f75--layer5.netlify.app

**Comment by leecalcote** [comment]
@ayushthe1 please review.

File: src/sections/General/Navigation/index.js [hunk 1]
```diff
@@ -238,10 +238,20 @@ const Navigation = () => {
...
+                                function (){
+                                  changeDropdownState();
+                                  closeDropDown();
+                               }
+                              } className="mobile-sub-menu-item" rel="noreferrer">
                                 {subItems.name}
                               </a>
-                              : <Link to={subItems.path} onClick={changeDropdownState} className="mobile-sub-menu-item" activeClassName="nav-link-active">{subItems.name}</Link>
+                              : <Link to={subItems.path} onClick={
+                                function (){
```
Comment (Udit-takkar) [inline src/sections/General/Navigation/index.js:250]: NIT: use arrow functions here like

onClick={ ( ) => {   
     changeDropdownState();
     closeDropDown();
  }
}

**Review by Udit-takkar — APPROVED** [review]
LGTM. works fine

Left one comment

**Comment by l5io** [comment]
🚀 Preview for commit 380493d033cfcfb157cec9dd95480bf5c0442b8e at: https://644c6273d6f8a330614c8a3b--layer5.netlify.app

**Review by Udit-takkar — APPROVED** [review]
LGTM

**Review by thoughtlessmind — APPROVED** [review]

**Comment by thoughtlessmind** [comment]
LGTM

**Comment by shibu-g** [comment]
@leecalcote do i need to update anything in this pr ? 
because i am waiting for this pr to close.

**Pull request merged**